examples/llm_inference/ios/InferenceExample/ConversationViewModel.swift

```diff
@@ -72,7 +72,9 @@ class ConversationViewModel: ObservableObject {
   }
 
   private func internalSendMessage(_ text: String) async {
-    chatTask?.cancel()
+    if busy {
+      chatTask?.cancel()
```

Comment:
We don't support cancellation at the UI level - unfortunately, you have to show an error here (or block out submit button).

Reply:
I opted to disable the message submit button instead.